exhibitor-standalone/src/main/resources/buildscripts/standalone/gradle/build.gradle

```diff
@@ -1,7 +1,17 @@
+version = '1.0'
```

Comment:
This is for building the fat jar (see [this wiki page](https://github.com/Netflix/exhibitor/wiki/Building-Exhibitor) and [the pom equivalent](https://github.com/Netflix/exhibitor/wiki/Building-Exhibitor).) 

What is the appropriate way to get the version number?